functions/utils/useCache.ts

```diff
@@ -0,0 +1,15 @@
+export async function getCache(request: string, name: string) {
```

Comment:
These would be easier to test and to read if you exposed a `Cache` singleton instead, with `get` and `put` methods.

Reply:
Like just returning `await caches.open(name)` and then allowing the functions to call the `get` and `put` methods?